Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -757,7 +757,7 @@ - (CGSize)cellSizeForViewWidth:(int)viewWidth contentWidth:(int)contentWidth
     OWSAssert(self.viewItem);
     OWSAssert([self.viewItem.interaction isKindOfClass:[TSMessage class]]);
 
-    const int maxMessageWidth = (int)floor(contentWidth * 0.7f);
+    const int maxMessageWidth = (int)floor(contentWidth * 0.8f);
```

Comment:
Incidentally, I've been thinking of moving this calculation of max message width into `OWSMessageCell`.

Reply:
Scratch that, I see that I already did. =P